app/org/sagebionetworks/bridge/models/accounts/FPHSExternalIdentifier.java

```diff
@@ -0,0 +1,23 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import org.sagebionetworks.bridge.dynamodb.DynamoFPHSExternalIdentifier;
+import org.sagebionetworks.bridge.json.BridgeTypeName;
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@BridgeTypeName("ExternalIdentifier")
+@JsonDeserialize(as=DynamoFPHSExternalIdentifier.class)
+public interface FPHSExternalIdentifier extends BridgeEntity {
+
+    public static FPHSExternalIdentifier create(String externalId) {
+        return new DynamoFPHSExternalIdentifier(externalId);
+    }
+    
+    public String getExternalId();
+    public void setExternalId(String externalId);
+    
+    public boolean getRegistered();
```

Comment:
Minor: Should this be isRegistered() instead of getRegistered()? (You can use a Jackson annotation to map it to the JSON attribute, if Jackson doesn't pick it up on its own.)